src/stream/src/executor/flow_control.rs

```diff
@@ -70,6 +73,29 @@ impl FlowControlExecutor {
                             }
                         }
                     }
+                    #[cfg(madsim)]
```

Comment:
Due to our offline discussion, here the reason why `madsim` doesn't support `governor::RateLimiter` is because it spawns system thread.

In this case, I think the best approach is to implement a rate limiter by ourselves or use another 3rd-party lib compatible with tokio 🤔

Reply:
A trivial implementation: acquire semaphore and spawn a task which sleeps for 1 sec before giving the permits back. 😄

Reply:
>  I think the best approach is to implement a rate limiter by ourselves

Resolved: 
- https://github.com/risingwavelabs/risingwave/pull/13144/commits/820666e7c7316f217644aef11a6160ec3511c3bb
- https://github.com/risingwavelabs/risingwave/pull/13144/commits/dc9b33d714c0d2a3de258814be637b54179aad4b